### PR TITLE
Create ko.json

### PR DESCRIPTION
**Pull request by DDinghoya**

### CODE REVIEW

File: src/plugins/i18n/locales/ko.json [hunk 1]
```diff
@@ -0,0 +1,944 @@
...
+          "background": "배경",
+          "transparency": "투명성"
+        },
+        "window": {
+          "maximize": "시작 시 최대화",
+          "quit": "닫을 시 나가기",
+          "showTrayIcon": "트레이 아이콘 표시"
+        },
+        "tabs": {
+          "newTabSwitch": 새 탭으로 전환",
```
Comment (staniel359) [inline src/plugins/i18n/locales/ko.json]: @DDinghoya Missed quotes here.

**Review by staniel359 — CHANGES_REQUESTED** [review]
@DDinghoya Please also translate [this file](https://github.com/staniel359/muffon/blob/main/electron/plugins/i18n/locales/en.json).

**Comment by staniel359** [comment]
@DDinghoya Also please join [our Discord server](https://discord.gg/QAmJq5EpEb).
All translators are there.

**Comment by staniel359** [comment]
@DDinghoya Also please translate [this one](https://github.com/staniel359/muffon-api/blob/main/config/locales/en.yml) (from related project).

File: src/plugins/i18n/locales/ko.json (hunk 1 above)
Comment (DDinghoya) [reply src/plugins/i18n/locales/ko.json]: Added it. Done :)

**Review by DDinghoya — COMMENTED** [review]
Edited 539 lines, Added quotes

**Comment by staniel359** [comment]
@DDinghoya Please notice my comments above.

**Comment by xyloflake** [comment]
> Edited 539 lines, Added quotes

You're not allowed to make translations before you have prior permissions from our end @DDinghoya 

While we do appreciate your efforts, we humbly ask you to comply by our rules as indicated by @staniel359 above. Your pr will NOT be merged if you don't take specific rules from us first in the discord.

**Comment by xyloflake** [comment]
Converted to draft because the pr isn't ready for merging. I request @DDinghoya to only open the pr after complying by our rules and when the pr is ready to be merged

**Comment by DDinghoya** [comment]
> @DDinghoya Also please join [our Discord server](https://discord.gg/QAmJq5EpEb). All translators are there.

Ok, I will join muffon discord!

**Pull request closed without merging**